compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -168,13 +169,13 @@ void addSimplePackBasedPassPipeline(OpPassManager &pm,
   modulePassManager.addPass(createCanonicalizerPass());
   modulePassManager.addPass(createCSEPass());
 
-  // Second level packing and bufferize to allocation
+  // Second level packing and only promote the result to local memory
   packOptions.packLevel = 1;
   packOptions.usePassPipeline = AIEPassPipeline::SimplePackPipeline;
   modulePassManager.addNestedPass<func::FuncOp>(
       createAMDAIEPackAndTransposePass(packOptions));
   bufferizeOptions.memorySpace = 2;
-  bufferizeOptions.bufferizeLevel = -1;
+  bufferizeOptions.bufferizeLevel = 1;
```

Comment:
Oh what does this do.

Reply:
This bufferizeLevel is used to decide what op we'd want to bufferize to a new allocation. For pad pipeline and this adapted simple pack pipeline, we don't want to promote result and inputs at the same time. Here's the source code for what each value stands for https://github.com/nod-ai/iree-amd-aie/blob/main/compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEBufferizeToAllocation.cpp#L48